Exercise_6/MainActivity.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.App;
+using Android.Widget;
+using Android.OS;
+using Android.Support.V7.Widget;
+using Exercise_6.Adapters;
+using Exercise_6.Models;
+
+namespace Exercise_6
+{
+    [Activity(Label = "Exercise_6", MainLauncher = true)]
+    public class MainActivity : Activity
+    {
+        [InjectView(Resource.Id.recyclerView)] private RecyclerView recyclerView;
+
+        private List<object> callSmsObjects;
+
+        private CallSmsAdapter adapter;
+        protected override void OnCreate(Bundle savedInstanceState)
```

Comment:
An empty line between 
```C#
private CallSmsAdapter adapter;
```
and
```C#
protected override void OnCreate(Bundle savedInstanceState)
```
is recommended

Reply:
Done